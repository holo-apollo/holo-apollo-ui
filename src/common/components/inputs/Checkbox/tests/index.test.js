import React from 'react';
import { mount } from 'enzyme';
import renderer from 'react-test-renderer';

import Checkbox from '../index';

describe('Checkbox', function() {
  it('should render normally', function() {
    const wrapper = mount(<Checkbox name="name" label="Label" />);
    expect(wrapper.exists()).toBe(true);
  });

  it('should match snapshot', function() {
    const tree = renderer
      .create(<Checkbox name="name" label="Label" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
