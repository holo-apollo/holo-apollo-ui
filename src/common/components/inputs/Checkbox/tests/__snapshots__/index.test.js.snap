// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox should match snapshot 1`] = `
<label
  className="styled__StyledLabel-sc-93nche-0 hZnQSg"
>
  <input
    className="styled__StyledCheckbox-sc-93nche-1 jNdYEo"
    name="name"
    size={20}
    type="checkbox"
  />
  Label
</label>
`;
